portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -33,28 +39,47 @@
 public class DataServlet extends HttpServlet {
 
   private static final String COMMENT_ENTITY_KEY = "Comment";
-  private static final String COMMENT_PROPERTY_NAME = "comment";
-  private static final String TIME_PROPERTY_NAME = "dateTime";
+  private static final String COMMENT_PROPERTY_NAME = "userComment";
+  private static final String TIME_PROPERTY_NAME = "dateTimeCreated";
   
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query(COMMENT_ENTITY_KEY)
+                  .addSort("dateTimeCreated", SortDirection.DESCENDING);
```

Comment:
Replace `"dateTimeCreated"` with `TIME_PROPERTY_NAME`. If you reference a nonexistent field, you get a compiler error. If you misspell a String, you won't know anything is wrong until runtime at best--and in more complicated code, the error can be hidden by other parts of your application logic.

This is an anti-pattern known as a "magic string". See [this StackOverflow post](https://softwareengineering.stackexchange.com/questions/365339/what-is-wrong-with-magic-strings) for a more detailed breakdown.